pg_backup_api/version.txt

```diff
@@ -1 +1 @@
-2.1.0
+2.2.0
```

Comment:
Shouldn't version be `2.2.1` in this case?
I mean, we are only shipping a bugfix.